test/uk/gov/hmrc/cgtpropertydisposals/controllers/returns/SubmitReturnsControllerSpec.scala

```diff
@@ -237,16 +237,16 @@ class SubmitReturnsControllerSpec extends ControllerSpec {
 
         "there is no reference id for a representee" in {
 
-            val requestBody = sample[SubmitReturnRequest].copy(
-              checkYourAnswerPageHtml = B64Html(""),
-              completeReturn = sample[CompleteSingleDisposalReturn].copy(
-                representeeAnswers = Some(
-                  sample[CompleteRepresenteeAnswers].copy(id = NoReferenceId)
-                )
+          val requestBody = sample[SubmitReturnRequest].copy(
+            checkYourAnswerPageHtml = B64Html(""),
```

Comment:
can we get a generator for `B64Html` or will a random string break the tests?

Reply:
it'll break it I think cos it'll try to base64 decode it in the code somewhere

Reply:
I think it shouldn't because what's there at the moment is not valid HTML anyway and we're not doing integration tests here so it should be fine? one way to find out....

Reply:
it'll probably break around here:
https://github.com/hmrc/cgt-property-disposals/blob/master/app/uk/gov/hmrc/cgtpropertydisposals/controllers/returns/SubmitReturnsController.scala#L89

Reply:
we decode it in the controller to be able to sanitise the html in it

Reply:
pushed up some code to allow B64HTML gen

Reply:
thanks, looks good